src/validators.js

```diff
@@ -1,20 +1,44 @@
 
 const assert = require('assert');
 
+
 function validateApiKey(value) {
-    const typeError = new TypeError('ImgixAPI.settings.apiKey must be passed a string');
+    const invalidApiKeyError = new TypeError('ImgixAPI.settings.apiKey must be passed a string');
     const legalKey = /[0-9a-f]{64}/;
     const legalKeyError = new TypeError(`${value} does not match a legal apiKey structure`);
 
-    assert(typeof value === 'string', typeError);
+    assert(typeof value === 'string', invalidApiKeyError);
     assert(legalKey.exec(value), legalKeyError);
 };
 
 function validateOpts(options) {
     validateApiKey(options.apiKey);
 };
 
+function validateBody(body) {
+    const invalidBodyError = new TypeError('The request body must be of type JSON or Buffer');
```

Comment:
"JSON" isn't really a type. How about this instead:

```suggestion
    const invalidBodyError = new TypeError('The request body must a valid JSON object or a Buffer');
```